Development-Guide/Git-Guides-and-Standards/Branching-Standards.adoc

```diff
@@ -12,9 +12,9 @@ As a result, *no one should ever force push to the master branch!*
 User branches are the most common kind of branch, that an individual user opens to work on a task. 
 These branches should be short lived as they should only be for small items.
 
-The naming for user branches will be along the lines of `user/<username>/<Issue Number>-Issue-Name`.
+The naming for user branches will be along the lines of `user/<username>/<JIRA-Issue Number>`.
 
-For example, `user/bill-luu/98-Improve-Git-Process`
+For example, `user/bill-luu/SFT-98-Improve-Git-Process`
```

Comment:
The format you listed is `user/<username>/<JIRA>-<Issue Number>`, so I assume you forgot to remove the name in the example. 
If this is not the case then my question is - why do we need the name when we have the issue number, where all information to the issue can be found there?